ansible/couchdb.yml

```diff
@@ -8,12 +8,12 @@
   - name: check if db_local.ini exists?
     tags: ini
     stat: path="{{ playbook_dir }}/db_local.ini"
-    register: db
```

Comment:
renaming of this variable is nessecary, because otherwise the `db` part in `group_vars/all` will be evaluated. This fails, because the db_local.ini does not exist yet.